src/node_file.cc

```diff
@@ -1568,6 +1568,38 @@ static void Rename(const FunctionCallbackInfo<Value>& args) {
   }
 }
 
+static void RenameSync(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  Isolate* isolate = env->isolate();
+
+  CHECK_GE(args.Length(), 2);
```

Comment:
```suggestion
  CHECK_EQ(args.Length(), 2);
```

Reply:
In `node_file.cc`, that's because a lot of methods accept different number of arguments for different cases.

Reply:
Yeah, you're right. Updated.